templates/unattended-upgrades.erb

```diff
@@ -23,6 +23,15 @@ Unattended-Upgrade::Package-Blacklist {
 <% end -%>
 };
 
+<% if @whitelist -%>
```

Comment:
Isn't this always true? I think an empty array also evaluates to true. Should it also check for an empty array?